cmd/main.go

```diff
@@ -83,7 +82,7 @@ func main() {
 				return err
 			}
 
-			db, err = datastore.New(cfg)
+			db, err = mongo.New(cfg)
```

Comment:
shouldn't the choice of mongo here be based on the specs in cfg?

Reply:
Yes, but I'll add the switch later Everything would be ready by tomorrow. I don't want to break implementation which is why I defaulted to mongo

Reply:
ohh okay, i get. Nice one.